_data/locales/en.yml

```diff
@@ -131,11 +131,26 @@ technologies:
       collab plugin, which makes it much easier for new users to get 
       started in packaging. It also allows you to work on a single 
       package with the same name from multiple projects, where they 
-      will be updated simultanously.     
+      will be updated simultanously. 
    software:
       trollolo:
         name: trollolo
         url: https://github.com/openSUSE/trollolo
       osc-plugin-collab:
         name: osc-plugin-collab
         url: https://github.com/openSUSE/osc-plugin-collab
+about_title: About openSUSE
+about_description: |
+  openSUSE is a Free Software project created around the openSUSE Linux 
```

Comment:
I would just say around open source in general, we have stated our distributions and technologies above already, don't really see much of a reason to repeat that